### PR TITLE
update 06-08-2017

### PR DESCRIPTION
**Pull request by Drapegnik**

### CODE REVIEW

File: tips/06-08-2017/React.xml [hunk 1]
```diff
@@ -316,4 +316,11 @@
...
   </template>
-</templateSet>
+  <template name="onc" value="onClick={$END$} " description="onClick event props" toReformat="true" toShortenFQNames="true">
+    <context>
+      <option name="JAVA_SCRIPT" value="true" />
+      <option name="JS_EXPRESSION" value="false" />
+      <option name="JS_STATEMENT" value="false" />
+    </context>
+  </template>
+</templateSet>
```
Comment (jakwuh) [inline tips/06-08-2017/React.xml]: missing line break

File: tips/06-08-2017/Readme.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
-#### Subscribe to our [Telegram channel](https://t.me/dailytip) to get daily updates
```
Comment (jakwuh) [inline tips/06-08-2017/Readme.md:1]: Removing subscribe teaser? Why?

File: tips/06-08-2017/Readme.md [hunk 3]
```diff
@@ -1,14 +1,11 @@
-#### Subscribe to our [Telegram channel](https://t.me/dailytip) to get daily updates  
+# live templates
+useful shortcuts for WebStorm [live templates feature](https://www.jetbrains.com/help/webstorm/live-templates-2.html)
 
----
+<p align="center">
```
Comment (jakwuh) [inline tips/06-08-2017/Readme.md]: Setting [width=700] will create inconveniences for mobile readers. This should be:
```live template demo```

File: tips/06-08-2017/Readme.md [hunk 4]
```diff
@@ -545,8 +465,3 @@ name | expression | default
     // FIXME(@$WHO$): $TEXT$
 $END$
 ```
-
-***
-
-Special credits to [@drapegnik](https://github.com/Drapegnik) for a useful set of live templates.
```
Comment (jakwuh) [inline tips/06-08-2017/Readme.md]: Removing footer teaser? Why?

**Review by jakwuh — CHANGES_REQUESTED** [review]

File: tips/06-08-2017/React.xml (hunk 1 above)
Comment (Drapegnik) [reply tips/06-08-2017/React.xml]: this file is generated by WebStorm, are you sure it's needed to fix it every time?

File: tips/06-08-2017/Readme.md (hunk 2 above)
Comment (Drapegnik) [reply tips/06-08-2017/Readme.md:1]: oh, sorry, it's accidently due to copy pasting 😮

**Comment by Drapegnik** [comment]
@jakwuh, hey!

**Comment by jakwuh** [comment]
Sorry for a small delay :) Thank you for updating your PR 👍

**Pull request merged**